test/Serialization.purs

```diff
@@ -83,20 +91,35 @@ suite = do
         txo <- convertTxOutput txOutputFixture1
         let bytes = toBytes (asOneOf txo)
         byteArrayToHex bytes `shouldEqual` txOutputBinaryFixture1
-      test "Transaction serialization #1" $ liftEffect do
-        tx <- convertTransaction txFixture1
-        let bytes = toBytes (asOneOf tx)
-        byteArrayToHex bytes `shouldEqual` txBinaryFixture1
-      test "Transaction serialization #2 - tokens" $ liftEffect do
-        tx <- convertTransaction txFixture2
-        let bytes = toBytes (asOneOf tx)
-        byteArrayToHex bytes `shouldEqual` txBinaryFixture2
-      test "Transaction serialization #3 - ada" $ liftEffect do
-        tx <- convertTransaction txFixture3
-        let bytes = toBytes (asOneOf tx)
-        byteArrayToHex bytes `shouldEqual` txBinaryFixture3
-      test "Transaction serialization #4 - ada + mint + certificates" $
-        liftEffect do
-          tx <- convertTransaction txFixture4
-          let bytes = toBytes (asOneOf tx)
-          byteArrayToHex bytes `shouldEqual` txBinaryFixture4
+      serializeTXs
+        [ txFixture1 /\ txBinaryFixture1
+        , txFixture2 /\ txBinaryFixture2
+        , txFixture3 /\ txBinaryFixture3
+        , txFixture4 /\ txBinaryFixture4
+        , txFixture5 /\ txBinaryFixture5
+        ]
+    group "Transaction Roundtrips" $ do
+      txSerializedRoundtrip
```

Comment:
Same as above - undo the grouping, it makes debugging harder.

Reply:
Fixed